base/Makefile

```diff
@@ -57,6 +57,7 @@ else
 	@echo "const USE_GPL_LIBS = false" >> $@
 endif
 	@echo "const libllvm_version_string = \"$$($(LLVM_CONFIG_HOST) --version)\"" >> $@
+	@echo "const libllvm_name = \"$(LLVM_SHARED_LIB_NAME)\"" >> $@
```

Comment:
ci error:
```
Error in testset libLLVM_jll:
Error During Test at V:\julia\test\testdefs.jl:21
  Got exception outside of a @test
  LoadError: InitError: could not load library ".dll"
  The specified module could not be found.
```

local test
```sh
$ make -C base/ print-LLVM_SHARED_LIB_NAME
LLVM_SHARED_LIB_NAME=

$ make print-LLVM_SHARED_LIB_NAME
LLVM_SHARED_LIB_NAME=libLLVM-14jl
```

Perhaps this variable needs to be imported first.
https://github.com/JuliaLang/julia/blob/0a9abc1919a69b7b4d668e3f841b53b186448e41/Makefile#L3-L4

Reply:
Thanks!